topics/single-cell/tutorials/scatac-preprocessing-tenx/tutorial.md

```diff
@@ -0,0 +1,675 @@
+>    {: .comment}
+>    > <comment-title>BED / encode narrowPeak format</comment-title>
+>    > If you are not familiar with BED format or encode narrowPeak format, see the [BED Format](https://genome.ucsc.edu/FAQ/FAQformat.html)
+>    {: .comment}
+>
+{: .hands_on}
+
+# Count matrix creation
+
+Now we have ATAC fragments with cell barcode information as well as the peaks to create a scATAC count matrix. In the context of ATAC-seq, the peaks are also often called features. But the term *feature* is a generic term that denotes a *peak* in the case of scATAC-seq data and a *gene* in case of scRNA-seq data. So the count matrix we build is similar to that of scRNA-seq data with the exception that here we have open chromatin regions instead of genes. In this tutorial, both the terms *feature* and *peak* indicate an *open chromatin region*. 
+
+For count matrix creation, we will use **Build count matrix** from **EpiScanpy** tool suite. It takes fragments file and the peaks in either BED format or directly a narrowPeak file of **MACS2** output. Regardless of file type, the tool only considers the first 3 columns which contain chromosome id, start and end positions of the peak regions. Parameters like `--call-summits` from our MACS2 run can result in different peaks with the same peak boundaries. It is always a  good practice to remove duplicate peak regions so that our final count matrix will have unique features.
+
+
+> <hands-on-title>Extract unique peak regions</hands-on-title>
+>
+> 1. {% tool [Unique occurrences of each record](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sorted_uniq/1.1.0) %} with the following parameters:
+>    - *"File to scan for unique values"*: `narrow Peaks` (output of **MACS2** {% icon tool %})
+>    - In *"Advanced Options"*: `Show Advanced Options`
+>        - *"Column start"*: `Column: 1`
+>        - *"Column end"*: `Column: 3`
+>
+> > <question-title></question-title>
+> >
+> > 1. How many peak regions were duplicated?
+> >
+> > > <solution-title></solution-title>
+> > >
+> > > 1. There were initially 1064 regions in the `narrow Peaks` file. Now there are 891 regions after deduplication. More than 15% (173) of regions have the same peak boundaries. 
+> > >
+> > {: .solution}
+> >
+>   {: .question}
+{: .hands_on}
+
+## AnnData
+
+The most common format, called [`AnnData`](https://anndata.readthedocs.io/en/stable/), stores the matrix as well as gene and cell annotations in a concise, compressed and extremely readable manner:
+
+![Anndata format]({% link topics/single-cell/images/scrna-pre-processing/tenx_anndata.svg %} "<code>AnnData</code> format stores a count matrix <code>X</code> together with annotations of observations (i.e. cells) <code>obs</code>, variables (i.e. genes/peaks/features) <code>var</code> and unstructured annotations <code>uns</code>.")
```

Comment:
This image is too small, it is not possible to read the content. 

![Screenshot from 2023-04-24 11-13-30](https://user-images.githubusercontent.com/26678117/233953197-f34db641-7b00-487e-bdfa-fc000f9bdd41.png)

Reply:
I don't what is causing it. I used a png file instead of an SVG and now it renders a big image.